chamber/utils/json.py

```diff
@@ -0,0 +1,16 @@
+from django.core.serializers.json import DjangoJSONEncoder
+
+
+class ChamberJSONEncoder(DjangoJSONEncoder):
+
+    def default(self, val):
+        if isinstance(val, bytes):
+            return {
+                '__type__': bytes.__name__,
```

Comment:
This is strange, bytes does not have `__name__` attribute, did you mean `__class__.__name__`?

Reply:
```
Python 3.8.3 (default, May 17 2020, 18:15:42) 
[GCC 10.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> bytes.__class__.__name__
'type'
>>> bytes.__name__
'bytes'
>>> int.__class__.__name__
'type'
>>> int.__name__
'int'
```

Reply:
Oh, you are right, I thought you are accessing the value of type bytes, my bad.